package-lock.json

```diff
@@ -1,8 +1,12571 @@
 {
   "name": "sls",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
@slsys can you check if it builds localy and works? I'm a little bit change the package-lock.json (moved to new version of npm) and I don't know if you are using new one... 
I can revert this change =)